### ISSUE TITLE
Trouble Following the Documentation

### ISSUE THREAD

**Issue by michaelgrilo** [Lujeni/matterllo#10]
Without having an extensive knowledge of Mattermost and/or Flask, the documentation has been very hard to follow. The trouble starts on `requirements.html`. No information is provided about installing `pip`; not too big of a deal. The generating an incoming webhook was easy enough. Then in the next section, it is not very clear what I will be doing with the Trello key and token.

Under `settings.html`, I'm at a loss as to what the first step is. I understand I need to generate a config file, but it is not clear to me where this file should be located and what it should be named (later on I see that it should be named config.yaml, but why not have this in the same place as the rest of the config information?). 

I looked through all of the output of `pip install matterllo` and I just cannot sort out which directory any of this is supposed to be happening in. 

Any help making the documentation easier to understand would be truly appreciated.

**Comment by Lujeni** [comment]
Hello,

Thanks for your feedback !

I wil try to summarize your issue and create a pull request with a check list to improve the documentation.

**Comment by Lujeni** [comment]
Hi,

I decided to create a real admin interface to facilitate the `matterllo` integration. #12 
It's will be much easier to create the bridge between trello and mattermost.

I will update the documentation after this feature.

**Comment by lifeboy** [comment]
Is there progress with the admin interface?  I too struggle to make sense of the installation process since I'm not a python developer.  I suppose one can study all the background info to be able to understand what should be happening, but I really don't want to have to do that, I just would like to integrate mattermost with trello.

**Comment by GhostLyrics** [comment]
@lifeboy I've since updated the documentation on a few spots. I'd ask you kindly to state where things are not as easy as they could be and I'll try my best to improve there so @Lujeni can focus on coding.

**Comment by Lujeni** [comment]
@lifeboy Yep i am on it but i am a litle bit slow.

Another works for the ansible project take all my free time right now.

Feel free to ping me on IRC or another chat (choose one :)) for your installation.

**Comment by akmet** [comment]
Hello,
i just found this extension and for me it is not clear what exactly is meant by callback_url and how to configure it.. Hope you can explain it with a little bit more detail.

When you need help creating the administration interface feel free to post on how we can help you!

**Comment by Lujeni** [comment]
Admin interface will be release this week.

**Comment by Lujeni** [comment]
I closed this issue. The complete refactoring is done.

Feel free to ping me if the setup is always a mess (i will re-open ths issue).
